tensorboard/components/tf_data_selector/tf-data-select-row.ts

```diff
@@ -31,6 +31,17 @@ Polymer({
       }),
     },
 
+    enabled: {
```

Comment:
I could be missing something, but is this actually functional in this PR?  Nothing seems to be hooked to this property except the checkbox so AFAICT it doesn't actually control the selection.

Speaking of which, is the thought that checkbox enable/disable state will also be persistent via the URL hash?  Or is that transient state only?

Reply:
Correct. It is there visually but does not actually do anything useful. Eventually, we will persist it in the URL (via hash) as you indicated :)